### PR TITLE
Make max amount of peers configurable

### PR DESCRIPTION
**Pull request by dshulyak**

### CODE REVIEW

**Review by divan — APPROVED** [review]
LGTM. But what do we need it for? Is it for whisper cluster testing tasks?

**Comment by dshulyak** [comment]
@divan Yes, i feel like 25 peers are generating too much unrealistic spam. I will use it with this tool https://github.com/status-im/status-go/pull/609

Also, i want to see how pattern will change when there will be 2-3 hops between peers.

File: cmd/statusd/main.go [hunk 1]
```diff
@@ -33,6 +33,7 @@ var (
 	lesEnabled     = flag.Bool("les", false, "LES protocol enabled (default is disabled)")
 	whisperEnabled = flag.Bool("shh", false, "Whisper protocol enabled (default is disabled)")
 	swarmEnabled   = flag.Bool("swarm", false, "Swarm protocol enabled")
+	maxPeers       = flag.Int("maxpeers", 25, "maximum amount of p2p peers (includes all protocols)")
```
Comment (adambabik) [inline cmd/statusd/main.go]: Small gramma thing I believe: `peers` are countable so it should be `maximum number of p2p peers (including all protocols)`.

**Review by adambabik — APPROVED** [review]
It will be definitely helpful for testing some hypothesis, for instance, linear relationship between a number of cached Whisper messages (and network usage) vs a number of peers.

**Review by mandrigin — APPROVED** [review]

**Pull request merged**